login/db/CreateUserResponce.js

```diff
@@ -1,4 +1,4 @@
-const UserResponse = require('./UserResponse.js');
+const UserResponse = require('./db_objects/UserResponse.js');
```

Comment:
UserResponse is more api related, I would put it into the same folder where you have your controller.